internal/node/launcher.sh

```diff
@@ -142,10 +142,16 @@ fi
 
 # Export the location of the runfiles helpers script
 export BAZEL_NODE_RUNFILES_HELPER=$(rlocation "TEMPLATED_runfiles_helper_script")
+if [[ "${BAZEL_NODE_RUNFILES_HELPER}" != /* ]]; then
```

Comment:
Does this account for windows?

Reply:
I believe so as the this in an .sh script so even Windows absolute paths start with `/C/path/to/file`. Will fire up my Windows machine to be sure however.

Reply:
Oh right, I didn't know that, that's super nice if your'e right

Reply:
Looks like you were right after all. $(rlocation) returns `C:/path/to/file` for absolute paths on Windows.

Reply:
ah shoot, oh well, good to know!